src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -54,7 +54,7 @@ You can include your DSN directly in these two files, or provide it in either of
 
 ## Extend Default Webpack Usage for Source Maps
```

Comment:
Good change. Title is still confusing though, I think you've missed it

Reply:
Yep, updated 👍.